templates/cis2-nft/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[package]
+name = "cis2_nft"
```

Comment:
Should this not be parametrized, i.e., a placeholder, and then substituted by cargo init?